measure.rb

```diff
@@ -2981,6 +2981,8 @@ def self.add_airflow(runner, model, building, weather, spaces)
       building.elements.each("BuildingDetails/Enclosure/Attics/Attic[AtticType/Attic[Vented='true']]") do |vented_attic|
         vented_attic_values = HPXML.get_attic_values(attic: vented_attic,
                                                      select: [:vented_attic_sla, :vented_attic_constant_ach])
+        next if vented_attic_values[:vented_attic_sla].nil? and vented_attic_values[:vented_attic_sla].nil? # skip additional attic elements
```

Comment:
A potential bug here when the last element in the loop is an additional element, then the useful information in another element might be ignored because `vented_attic_sla` and `vented_attic_const_ach` are eventually assigned to be nil.

Reply:
```suggestion
        next if vented_attic_values[:vented_attic_sla].nil? and vented_attic_values[:vented_attic_constant_ach].nil? # skip additional attic elements
```

Reply:
Sorry for this silly error 🤦‍♀ Good catch!